app/src/main/java/ch/epfl/sdp/lobby/global/GlobalLobbyFragment.kt

```diff
@@ -0,0 +1,45 @@
+package ch.epfl.sdp.lobby.global
+
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import ch.epfl.sdp.databinding.FragmentGlobalLobbyBinding
+
+class GlobalLobbyFragment : Fragment() {
+
+    private lateinit var binding: FragmentGlobalLobbyBinding
```

Comment:
This should be nullable and set to null in onDestroyView
Source: https://developer.android.com/topic/libraries/view-binding#fragments

Reply:
will do